### ISSUE TITLE
Massive server performance drop

### ISSUE THREAD

**Issue by Zentheon** [dvitski/direbats#12]
Heyo, been using this mod for quite a while. I somehow never actually noticed until today, but with direbats loaded and having them spawn naturally in the world, the server performance can take huge dips. It seems somewhat situational, but on average having ten bats in the world at once can raise MSPT by two or even three times as without any. 
![Screenshot from 2020-12-23 20-26-09](https://user-images.githubusercontent.com/62160456/103057602-30ec4e00-455d-11eb-98af-6e96d69c612c.png)
![Screenshot from 2020-12-23 20-23-15](https://user-images.githubusercontent.com/62160456/103057610-38abf280-455d-11eb-8da4-6904b1a0a1a6.png)
The topmost screenshot is taken with a few direbats in it, and the MSPT shown by carpet mod, which is the only other thing I have installed besides fapi and direbats in this example. The bottom shot is without the bats. 
This impact would get much more severe in other situations which I couldn't really rule out, sometimes causing the server to run over a hundred milliseconds behind average

**Comment by Zentheon** [comment]
Has this been fixed in 1.1.0? too lazy to check atm lol

**Comment by dvitski** [comment]
I personally never experienced a performance drop, so I can't really test it :P

**Comment by Zentheon** [comment]
Odd. I'll try it again tomorrow and see, since I've seen others on curseforge have the same issue.

**Comment by Zentheon** [comment]
Okay so I totally forgot about testing this out but uh, better late than never? Anyway, I did a number of tests between both releases and it does indeed seem like the performance impact has been alleviated. There was one time I got an odd, large lag spike on 1.1.0, but I'm ruling it out as a fluke. 

Other than that, I noticed that during the day, the bats will flicker in between an on fire state without taking damage or anything when exposed to sunlight. It looked pretty odd, but it seems too obvious to be a bug, so I was curious about what that was?
